web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -129,15 +129,16 @@ export class LayerDialogComponent implements OnDestroy {
       fields = fields.set(fieldId, field);
     });
     const formId = this.dataStoreService.generateId();
+    const layerColor = '#ff9131';
     const layer = new Layer(
       this.layerId,
-      this.layer?.color ? this.layer.color : '#ff9131',
+      this.layer?.color ? this.layer.color : layerColor,
```

Comment:
You should be able to use `this.layer?.color || DEFAULT_LAYER_COLOR` instead of ternary here.